src/api/app/views/webui/workflow_runs/index.html.haml

```diff
@@ -0,0 +1,54 @@
+- @pagetitle = 'Workflow Runs'
+
+.card
+  .card-body
+    %h3= @pagetitle
+    .accordion.pt-3#workflow-runs-accordion
+      - @workflow_runs.each do |workflow_run|
```

Comment:
We should at least display a message like _This token wasn't triggered yet._ if there are no workflow runs for a token. Otherwise, it looks like something is wrong/broken.

![workflow-runs](https://user-images.githubusercontent.com/1102934/142390375-0c92676b-c21e-4db8-b6c8-db5510e685a6.png)